CREATING_A_RELEASE.md

```diff
@@ -0,0 +1,58 @@
+# Creating a release
+
+Only collaborators in npm for **node-api-headers** can create releases. If you
+want to be able to do releases ask one of the existing collaborators to add
+you. If necessary you can ask the build Working Group who manages the Node.js
+npm user to add you if there are no other active collaborators.
+
+## Prerequisites
+
+Before to start creating a new release check if you have installed the following
+tools:
+
+* [Changelog maker](https://www.npmjs.com/package/changelog-maker)
+
+If not please follow the instruction reported in the tool's documentation to
+install it.
+
+## Publish new release
+
+These are the steps to follow to create a new release:
+
+* Open an issue in the **node-api-headers** repo documenting the intent to
+create a new release. Give people some time to comment or suggest PRs that
+should land first.
+
+* Update the version in **package.json** appropriately.
+
+* Update the [README.md][] to show the latest stable version of Node-API.
+
+* Generate the changelog for the new version using **changelog maker** tool.
+From the root folder of the repo launch the following command:
+
+```bash
+> changelog-maker --format=markdown
+```
+
+* Use the output generated by **changelog maker** to update the
+[CHANGELOG.md][] following the style used in publishing the previous release.
+
+* Add any new contributors to the "contributors" section in the
```

Comment:
I didn't find steps to create git tags for the release. Should we document that the tag should be created along with package.json version updates by `npm version patch` or `npm version minor`?

Reply:
The tag will be created by GitHub while creating new release.

Reply:
Thanks, SGTM.